tasks/deployment/unpause.ts

```diff
@@ -2,18 +2,29 @@ import { task } from 'hardhat/config'
 import { cliOpts } from '../../cli/defaults'
 import GraphChain from '../../gre/helpers/network'
 
-task('migrate:unpause', 'Unpause protocol and bridge')
+task('migrate:unpause', 'Unpause protocol (except bridge)')
   .addOptionalParam('addressBook', cliOpts.addressBook.description)
   .addOptionalParam('graphConfig', cliOpts.graphConfig.description)
   .setAction(async (taskArgs, hre) => {
     const graph = hre.graph(taskArgs)
     const { governor } = await graph.getNamedAccounts()
-    const { Controller, L1GraphTokenGateway, L2GraphTokenGateway } = graph.contracts
+    const { Controller } = graph.contracts
 
     console.log('> Unpausing protocol')
     const tx = await Controller.connect(governor).setPaused(false)
     await tx.wait()
 
+    console.log('Done!')
+  })
+
+task('migrate:unpause-bridge', 'Unpause bridge')
```

Comment:
Same change as https://github.com/graphprotocol/contracts/pull/717/files#diff-729e91d21c664fed317c8bccc772f1c5cce6a8a4d943d700359ad371302aef55 

which is pending merge after audit. I'd still make the change here and later we solve the conflict whenever it shows up.

Reply:
Oooh oops, sorry I missed that. But yeah, let's keep the change here so that we have e2e passing here.